### ISSUE TITLE
multiple sum aggregation

### ISSUE THREAD

**Issue by SujPer** [nicolaskruchten/jupyter_pivottablejs#57]
Hello Nicolas,

I hope you are well. I was taking a look through the widget and think its fantastic. I did however have a question around the seeming restriction of the 'value' fields. Am I correct in assuming that it only allows one value field when the aggregation type is 'sum'? If so, is there any recommended workaround for this please?

Thank you,

**Comment by nicolaskruchten** [comment]
Yes, the built-in sum aggregator accepts only one value, but in principle you might be able to build a sum operator that sums across multiple values at the PivotTable.js level and then fork this Jupyter integration to include that. It's not something I have any time to work on, but there are no technical obstacles :)
